src/android/HockeyApp.java

```diff
@@ -115,6 +180,55 @@ public void run() {
             return true;
         }
         
+        if (action.equals("feedbackModal")) {
+            final ArrayList<Uri> attachments = new ArrayList<Uri>();
+            final Activity context = cordova.getActivity();
+            boolean takeScreenshot = args.optBoolean(0);
+            if (takeScreenshot) {
+                Bitmap screenshot = getBitmap();
```

Comment:
Could we not use [this method](https://github.com/bitstadium/HockeySDK-Android/blob/develop/hockeysdk/src/main/java/net/hockeyapp/android/FeedbackManager.java#L341) to get the screenshot?